docs/tutorials-samples/targeting-core-for-new-libraries.md

```diff
@@ -0,0 +1,298 @@
+# Targeting with .NET Core Libraries
```

Comment:
I think you need to explicitly say at the top that this is a draft. Lots of this will be changing as we finish things up.